src/index.ts

```diff
@@ -78,41 +78,49 @@ export default function wrapMotoko(compiler: Compiler) {
     };
 
     // Function signatures for `mo.parseMotokoTyped()`
+    /* old API, doesn't support cache and recovery */
     function parseMotokoTyped(paths: string): ParseMotokoTypedResult;
     function parseMotokoTyped(paths: string[]): ParseMotokoTypedResult[];
+    /* new API: */
     function parseMotokoTyped(
         paths: string,
         scopeCache: ScopeCache,
+        enable_recovery?: boolean,
     ): [ParseMotokoTypedResult, ScopeCache];
     function parseMotokoTyped(
         paths: string[],
         scopeCache: ScopeCache,
+        enable_recovery?: boolean,
     ): [ParseMotokoTypedResult[], ScopeCache];
     function parseMotokoTyped(
         paths: string | string[],
         scopeCache?: ScopeCache,
+        enable_recovery?: boolean,
```

Comment:
Use camelCase for JS. So `enableRecovery.`

Reply:
done